assets/shaders/model/model_loading.geom.glsl

```diff
@@ -0,0 +1,17 @@
+#version 460 core
+
+layout (triangles) in;
+layout (triangle_strip, max_vertices = 3) out;
+
+in vec2 TexCoords[];
+out vec2 fragTexCoords;
+
+void main() {
+    for (int i = 0; i < 3; i++) {
+      gl_Position = gl_in[i].gl_Position;
+      fragTexCoords = TexCoords[i];
```

Comment:
Is this any different from passing it from the vertex shader?

Reply:
Not really, but I added the geom for testing stuff and left it because it can be useful for debuging